router/core/graphql_handler.go

```diff
@@ -164,14 +165,21 @@ func (h *GraphQLHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
-		preparedPlan = sharedPreparedPlan.(planWithExtractedVariables)
+		preparedPlan = sharedPreparedPlan.(planWithMetaData)
 	}
 
 	extractedVariables := make([]byte, len(preparedPlan.variables))
 	copy(extractedVariables, preparedPlan.variables)
 	requestVariables := operationContext.Variables()
 	combinedVariables := MergeJsonRightIntoLeft(requestVariables, extractedVariables)
 
+	/*
+		preparedPlan.schemaUsageInfo.OperationType.String()
```

Comment:
@StarpTech At this point, you can create a "report" and send it to the metrics collector, or do it async, or batch it. Up to you.

Reply:
ty